src/models/StyleSheet.js

```diff
@@ -105,6 +106,10 @@ export default class StyleSheet {
     return this.tags.map(tag => tag.toHTML()).join('')
   }
 
+  toElement() {
+    return this.tags.map(tag => tag.toElement())
```

Comment:
Hm fighting with Flow:

```javascript
toElement() {
    return this.tags.map((tag, index) => (
      React.cloneElement(
        tag.toElement(),
        { key: index },
      )
    ))
  }
```

Gives error:
```[flow] property `key` of object literal (Property not found in props of React element `style` See also: call of method `cloneElement`)```

Seems like Flow doesn't like adding props to existing elements: https://github.com/facebook/flow/issues/1606

Any idea how to get around that?